docs/requirements.txt

```diff
@@ -2,5 +2,6 @@ docutils
 m2r2
 nbsphinx
 sphinx-copybutton
-sphinx>=3.1.0
+sphinx==4.5.0
 xanadu-sphinx-theme==0.1.0
+traitlets==5.2.0
```

Comment:
What do we need `traitlets` for?

Reply:
This again, also broke sphinx, and is a dependency used by sphinx. However, yesterday, traitlets also released a new version which broke [everything](https://github.com/ipython/traitlets/issues/741), I believe it is fixed now, so perhaps it should be okay to remove now?

Reply:
Oh, I didn't realize that was the case! Since it was patched relatively quickly and we do not _directly_ depend on `traitlets`, I suggest we remove the requirement and let our direct dependencies handle any version constraints for it.